### PR TITLE
coverage figure and associated text

### PR DESCRIPTION
**Pull request by hardingnj**
supersedes #5

### CODE REVIEW

File: content/02.results.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+## Population Sampling
+
+
+## Whole Genome Sequencing and Alignment
+
+Mosquitoes were submitted to the Ag1000G consortium in 23 samplesets (ref to table).
+
+A total of XXXX individual mosquitoes were sequenced on either Illumina hi-seq (n=XXX) or Illumina 10X (n=XXX) to a target coverage of 30X.
```
Comment (alimanfoo) [inline content/02.results.md]: MLWH queries: https://github.com/malariagen/vector-ops/blob/master/src/python/query_mlwh.py

**Comment by hardingnj** [comment]
Ready for review, incomplete- but ready for draft.

File: content/02.results.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+668 samples were removed where sequencing was of insufficient quality to accurately call genotypes across the whole genome.
+
+Exclusions were due to poor coverage (n=410), potential contamination (n=229), and the autosomal vs X coverage ratio not following the expected bimodal distribution (n=29).
+
+Where technical replicates were available, we excluded 4 pairs (8 samples) with low genotype concordance. 
+
+Where pairs met the concordance threshold we excluded the lower quality sample.
+
+In total 407 samples in were excluded in favour of better quality samples, based on skewedness of the mean vs median.
```
Comment (cclarkson) [inline content/02.results.md:41]: "mean vs median" what? Also, not sure how you'd like this reviewed, but I think vs should be written _vs._

File: content/02.results.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+Samples were also screened pairwise within submission sets for unexpected pairs, though none were detected.
```
Comment (cclarkson) [inline content/02.results.md:43]: Unexpected replicated pairs? I think this might need a little clarification.

**Comment by cclarkson** [comment]
Hi @hardingnj, how would you like this reviewed, content, grammar, the whole nine yards? It looks good to me (couple of comments above).

**Pull request merged**